freud/box.pyx

```diff
@@ -292,6 +292,10 @@ cdef class Box:
         Args:
             vecs (:math:`\left(3, \right)` or :math:`\left(N, 3\right)` :class:`numpy.ndarray`):
                 Unwrapped vector(s).
+            out (:class:`numpy.ndarray` or :code:`None`):
+                The array in which to place the wrapped vectors. It must be of
+                dtype `np.float32` or larger. If ``None``, this function will
```

Comment:
"Or larger"? That doesn't seem quite right. If it's not `float32` then a new array is allocated, right? I think this should be `float32` only.